doc/faq/security.md

```diff
@@ -0,0 +1,30 @@
+## Security
+
+```
+16:39 < Erkan> well, I am not so sure about using VPS, tbh
+16:39 < Erkan> your hoster (or the agency showing him the judge's paper) can get your config
+```
+
+Suppose you want to [pull a snowden](http://en.wikipedia.org/wiki/Global_surveillance_disclosures_%282013%E2%80%93present%29) and release some sensitive documents. Suppose that for some reason you're relying on an experimental protocol (cjdns) to do so. `Erkan` is wondering about the wisdom of doing so.
+
+[cjdns](https://github.com/cjdelisle/cjdns) is big and complicated, and without a fair deal of experience with cryptographic tools you may have a hard time understanding what's safe and what is not.
+
+## If you're hosting a cjdns node on a VPS
+
+There are quite likely measures that you can take to encrypt your VPS's hard drive, and you should look into that, but suppose you haven't encrypted anything. In that case, yes, your VPS provider can probably just go and look at your private keys, and read your encrypted traffic.
```

Comment:
this is misleading, if you have a VPS, your provider can pause you for a moment and take a snapshot of your ram, then dump your keys. There's no way around this, people write papers about how to protect against this, in the end you always lose. However PFS should be taken into account. If you lose your cjdroute.conf file, it doesn't mean you lost all sessions prior because temp keys are used and destroyed. It does mean someone can impersonate you later but it also doesn't mean they can just begin decrypting your traffic at will. Keys are rotated when CA sessions timeout which is "random" but obviously they are all killed when cjdroute is restarted.